src/core/test/system/users/investor.sol

```diff
@@ -30,6 +30,7 @@ contract Investor is DSTest {
 
     function doSupply(uint amount) public {
         address tranche_ = address(operator.tranche());
+        assertEq(currency.balanceOf(address(this)), amount);
```

Comment:
why do they need to supply the entirely of their balance every time?

Reply:
Yes, I wanted to add greater than